tests/flow/tests_llapi.py

```diff
@@ -150,3 +150,13 @@ def test_dagrun_multidevice_resnet(env):
 
     ret = con.execute_command("RAI_llapi.DAG_resnet")
     env.assertEqual(ret, b'DAG resnet success')
+
+# @with_test_module
```

Comment:
If we are leaving it here for now as comment, at least should have an explanation for why is it commented...